pom.xml

```diff
@@ -94,19 +98,20 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
-        <!--
-        Added killbill-catalog and killbill-xmlloader to benefit from Catalog objects implementation
-         (only because the plugin is cheating is reading an existing xml configuration file).
-         In a real catalog plugin those would not be needed.
-        -->
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.kill-bill.billing</groupId>
             <artifactId>killbill-catalog</artifactId>
             <version>${killbill.version}</version>
+            <scope>test</scope>
```

Comment:
I see.. so the package is still used for the `TestConversions` to be able to generate a `StandaloneCatalog` and call `validate`. I think this is fine to this dep. for test purpose as you did.